Lib9c/Action/BattleArena.cs

```diff
@@ -223,7 +224,26 @@ public override IAccountStateDelta Execute(IActionContext context)
                 arenaInformation.ResetTicket(currentTicketResetCount);
             }
 
-            arenaInformation.UseTicket(ticket);
+            if (roundData.ArenaType != ArenaType.OffSeason && ticket > 1)
+            {
+                throw new ExceedPlayCountException($"[{nameof(BattleArena)}] " +
+                                                   $"ticket : {ticket} / arenaType : {roundData.ArenaType}");
+            }
+
+            if (arenaInformation.Ticket > 0)
+            {
+                arenaInformation.UseTicket(ticket);
+            }
+            else
+            {
+                for (var i = 0; i < ticket; i++)
+                {
+                    var ticketBalance = ArenaHelper.GetTicketPrice(roundData, arenaInformation, states.GetGoldCurrency());
+                    var arenaAdr = ArenaHelper.DeriveArenaAddress(roundData.ChampionshipId, roundData.Round);
```

Comment:
i think it can be called once. because address can't change in this loop. right?

Reply:
👍🏻 I fixed it 🙏🏻 @ipdae